docs/2.guide/2.directory-structure/1.components.md

```diff
@@ -150,14 +150,6 @@ To dynamically import a component (also known as lazy-loading a component) all y
 This is particularly useful if the component is not always needed. By using the `Lazy` prefix you can delay loading the component code until the right moment, which can be helpful for optimizing your JavaScript bundle size.
 
 ```html [pages/index.vue]
-<template>
-  <div>
-    <h1>Mountains</h1>
-    <LazyMountainsList v-if="show" />
-    <button v-if="!show" @click="show = true">Show List</button>
-  </div>
-</template>
-
 <script>
```

Comment:
Let's convert to composition api?

Reply:
I don't know this. It may be intentional to let the user know they can work with both modes. Because in the example below, the Composition API has been used.

Reply:
It would be worth ultimately supporting a toggle between modes but for now we can probably leave them as they are.